docs/01-app/05-api-reference/01-directives/use-client.mdx

```diff
@@ -43,6 +47,42 @@ export default function Counter() {
 }
 ```
 
+When using the `'use client'` directive, the props of the Client Components must be [serializable](https://react.dev/reference/rsc/use-client#serializable-types). This means the props need to be in a format that React can serialize when sending data from the server to the client.
+
+```tsx filename="app/components/counter.tsx" highlight={4} switcher
+'use client'
+
+export default function Counter({
+  useState /* ❌ Function is not serializable */,
```

Comment:
this example is a little odd b/c you would get useState from the React import. I'd pick a more conventional sounding function argument. like an even handler on `onClick` or something.